src/library/scala/collection/SeqView.scala

```diff
@@ -110,6 +110,7 @@ object SeqView {
     def length = (underlying.size - normN) max 0
     @throws[IndexOutOfBoundsException]
     def apply(i: Int) = underlying.apply(i + normN)
+    override def drop(n: Int): SeqView[A] = new Drop(underlying, this.n + n)
```

Comment:
I wonder why this wouldn't be in View.Drop rather than in `SeqView.Drop`?

Reply:
> Then we need an abstract factory type to create views.

It could be implemented separately in both classes. We do the same for other SeqView/IndexedView/MapView methods.